riocli/jsonschema/schemas/deployment-schema.yaml

```diff
@@ -94,6 +94,16 @@ definitions:
       subPath:
         type: string
         pattern: '^\/$|(^(?=\/)|^\.|^\.\.)(\/(?=[^/\0])[^/\0]+)*\/?$'
+      uid:
+        type: integer
+        default: 1000
```

Comment:
@pallabpain Does this mean if the user does not specify then the default will become 1000? If so I think this is not desired then we can remove the default field.